numpyro/examples/datasets.py

```diff
@@ -14,6 +14,7 @@
 import zipfile
 
 import numpy as np
+import scipy
```

Comment:
Could you move scipy to local function? We don't require scipy by default.

Reply:
I'll convert the format as you suggest below and remove scipy.